src/lsm/groove.zig

```diff
@@ -313,21 +305,6 @@ pub fn GrooveType(
         },
     });
 
-    // Verify no hash collisions between all the trees:
-    comptime var hashes: []const u128 = &.{_ObjectTree.hash};
-
-    if (has_id) {
-        const hash: []const u128 = &.{_IdTree.hash};
-        assert(std.mem.indexOf(u128, hashes, hash) == null);
-        hashes = hashes ++ hash;
-    }
-    inline for (std.meta.fields(_IndexTrees)) |field| {
-        const IndexTree = @TypeOf(@field(@as(_IndexTrees, undefined), field.name));
-        const hash: []const u128 = &.{IndexTree.hash};
-        assert(std.mem.indexOf(u128, hashes, hash) == null);
-        hashes = hashes ++ hash;
-    }
-
```

Comment:
This check was run at the groove scope, not the forest scope.
So it could detect collisions within a single groove, but it would miss hash collisions between multiple grooves.